packages/testlab/src/shot.ts

```diff
@@ -10,21 +10,25 @@
 
 // tslint:disable:no-any
 
+import * as express from 'express';
 import {IncomingMessage, ServerResponse} from 'http';
-import * as util from 'util';
-
 import {
+  Listener as ShotListener,
   RequestOptions as ShotRequestOptions,
   ResponseObject,
-  inject,
-} from 'shot';
+} from 'shot'; // <-- workaround for missing type-defs for @hapi/shot
```

Comment:
Correct me if I'm wrong, but are we not still importing from `shot` module? Does that mean we should still leave the `shot` dependency in `package.json`?

Reply:
This is importing **type declarations** only. The import statement is used by TypeScript compiler only, it resolves to importing types from `@types/shot` package and does not emit any JavaScript code.

The actual implementation of various Shot functions is always imported from `@hapi/shot` package, as can be seen elsewhere.

It's hacky, I hope somebody will publish new `@types/*` package soon and we will be able to get rid of this hack.

Reply:
> Are we not still importing from shot module?

No, we are not. This is just a way how to tell TypeScript to import type declarations from `@types/shot`, because it's not possible to write `import {...} from '@types/shot'`.

> Does that mean we should still leave the shot dependency in package.json?

No, because we are not importing from `shot` at runtime (nor compile time). If we kept `shot` in our dependencies, then we would still receive the deprecation warning, which would defeat the purpose of this change.

Reply:
Okay thanks for explaining, I see the `@types/shot` package 👍